test/support/.percy.yml

```diff
@@ -2,3 +2,7 @@ version: 1
 snapshot:
   widths: [375, 1280]
   min-height: 1024
+static_site:
+  base-url: /blog
```

Comment:
Wondering if all of these should be camel cased just like the other config options are:
- https://github.com/percy/percy-agent/blob/7e92365023f8869377ae4d9eb71f04c929941144/src/percy-agent-client/snapshot-options.ts
- https://github.com/percy/percy-agent/blob/7e92365023f8869377ae4d9eb71f04c929941144/src/services/agent-options.ts
- https://github.com/percy/percy-agent/blob/7e92365023f8869377ae4d9eb71f04c929941144/src/percy-agent-client/client-options.ts

Reply:
resolved with https://github.com/percy/percy-agent/pull/159/commits/2568731352e8dcdbd1095e19f163f9daec026ed5